CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Add model and API endpoints to manage a shared live media
 - Add component to display chat, viewers and apps during a live in a panel
+- Add `consumer_site` in JWT token
```

Comment:
In your commit message

- `s/For instance, a playlist for a giver context_id/For instance, a playlist for a given context_id/`

And you removed the last sentence `We've decided to simplify access to this information by adding it directly in the JWT token.` but I think you should keep it, WDYT ?